sdk/src/tx/whileValidTxSender.ts

```diff
@@ -15,8 +15,6 @@ import { IWallet } from '../types';
 
 const DEFAULT_RETRY = 2000;
 
-const VALID_BLOCK_HEIGHT_OFFSET = -150; // This is a bit of weirdness but the lastValidBlockHeight value returned from connection.getLatestBlockhash is always 300 blocks ahead of the current block, even though the transaction actually expires after 150 blocks. This accounts for that so that we can at least accuractely estimate the transaction expiry.
```

Comment:
RPCs are fixed now where lastValidBlockheight seems to contain the correct value now and no longer need these offsets.